backends/arm/quantizer/quantization_annotation/generic_annotator.py

```diff
@@ -53,6 +53,7 @@
     torch.ops.aten.tile.default,
     torch.ops.aten.flip.default,
     torch.ops.aten.cat.default,
+    torch.ops.aten.concatenate.default,
```

Comment:
Do we really see this op after export?

Reply:
This issue was noticed during internal testing after export and thus this PR was needed to fix this issue.